codeclimate-govet.go

```diff
@@ -1,30 +1,45 @@
 package main
 
-import "github.com/codeclimate/cc-engine-go/engine"
-import "strings"
-import "os"
-import "os/exec"
-import "strconv"
+import (
+	"github.com/codeclimate/cc-engine-go/engine"
+	"strings"
+	"os"
+	"os/exec"
+	"strconv"
+	"fmt"
+)
 
 func main() {
 	rootPath := "/code/"
 
 	config, err := engine.LoadConfig()
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error loading config: %v\n", err)
 		os.Exit(1)
 	}
 
 	analysisFiles, err := engine.GoFileWalk(rootPath, engine.IncludePaths(rootPath, config))
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error initializing: %v\n", err)
```

Comment:
do you have an example of how an error object looks when it gets interpolated into a string with the `%v` thing? I'm wondering if we should instead do `err.Error()` as we do a few lines down to get the error message as a string